cattle/plugins/host_info/main.py

```diff
@@ -0,0 +1,19 @@
+from cattle.plugins.host_info.memory import MemoryCollector
```

Comment:
Regarding the file name: "main.py"?

```from cattle.plugins.host_info.host_info import HostInfo``` just felt too redundant?

I'll buy it. ```from cattle.plugins.host_info.main import HostInfo``` seems reasonable.

I was worried about others attaching special meaning to a file called main.py, but I think I'm ok with this.